src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useMemo } from 'react'
+
+import type { FieldError } from 'react-hook-form'
+import { useSelector } from 'react-redux'
+
+import Button from 'components/Button'
+import { makeSelectSubCategories } from 'models/categories/selectors'
+
+import { StyledErrorMessage } from '../../../../../my-incidents/components/LoginForm/styled'
```

Comment:
Ideally, if this component is shared, it should be brought to a shared level. However, in this case it is just some padding, which you could also just add to the styled.ts here.

Reply:
Added it to the stylesheet. There is a generic component ErrorMessage that's now directly referenced.